### PR TITLE
Add Maintenance Mode Docs

### PR DESCRIPTION
**Pull request by crondinini**
### Description

Adds maintenance mode docs
image

### CODE REVIEW

File: motif-docs/pages/guides/experimental/maintenance-mode.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+<div className="drop-shadow-md rounded min-h-[200px]">
+
+<Image
+  className="margin-0 rounded"
+  height="200"
+  src="https://res.cloudinary.com/djp21wtxm/image/upload/v1666430995/i1600x983-LPvqk9j7nJVD_exnscl.png"
+/>
+</div>
+
+## How to enable it
```
Comment (flybayer) [inline motif-docs/pages/guides/experimental/maintenance-mode.mdx]:
```suggestion
## How To Enable It
```

File: motif-docs/pages/guides/experimental/maintenance-mode.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+You'll be prompted with a confirmation modal and once you confirm it, we'll start the process of enabling the mode for all the supported service within that Environment.
+
+Once the process has finished the services that have been set to Maintenance Mode will look like this:
+
+<Image
+ src="https://res.cloudinary.com/djp21wtxm/image/upload/v1666431532/i1600x576-MnKEoIcY9HmC_fh7cli.png"
+/>
+
+## How to disable it
```
Comment (flybayer) [inline motif-docs/pages/guides/experimental/maintenance-mode.mdx]:
```suggestion
## How To Disable It
```

File: motif-docs/pages/guides/experimental/maintenance-mode.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+To disable it and go back to the regular setup, you'll see the option in the same place where you enabled it, but this time you will not need to confirm.
+As soon as you click on the button, we'll revert the services back to the original configuration.
+
+## How it works
```
Comment (flybayer) [inline motif-docs/pages/guides/experimental/maintenance-mode.mdx]:
```suggestion
## How It Works
```

**Review by flybayer — COMMENTED** [review]
Docs are good! :)

To this image, can you (1) add an arrow callout to the button and (2) make it much smaller, basically zoom size similar to the dashboard.

image

File: motif-docs/pages/guides/experimental/maintenance-mode.mdx (hunk 1 above)
Comment (crondinini) [reply motif-docs/pages/guides/experimental/maintenance-mode.mdx]: @flybayer Just to share, I personally dislike capitalising every word on a title :D 
Also, I'm pretty sure it's considered "incorrect" to capitalise words like `to` and other prepositions/small words according to a few style guides.

**Review by flybayer — APPROVED** [review]

File: motif-docs/pages/guides/experimental/maintenance-mode.mdx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Go to the Environment where the services are located and click on the "Maintenance Mode" option within the Environment menu options.
+
+<div className="w-[300px]" >
+<img src="https://res.cloudinary.com/djp21wtxm/image/upload/v1666608119/i842x550-ik-QFoDGMUX4_n24afo.png" />
+</div>
+
+You'll be prompted with a confirmation modal and once you confirm it, we'll start the process of enabling the mode for all the supported service within that Environment.
+
+Once the process has finished the services that have been set to Maintenance Mode will look like this:
```
Comment (crondinini) [inline motif-docs/pages/guides/experimental/maintenance-mode.mdx]:
```suggestion
Once the process has finished, the services set to Maintenance Mode will look like this:
```

**Pull request merged**